packages/truffle-decoder/lib/allocate/storage.ts

```diff
@@ -23,6 +23,24 @@ export function storageLengthToBytes(size: StorageLength): number {
   }
 }
 
+export interface Range {
+  from: StoragePosition;
+  to?: StoragePosition;
+  length?: number;
+}
+
+export interface StoragePosition {
+  slot: Slot;
+  index: number;
+};
+
+export interface Slot {
+  key?: any; // TODO:
```

Comment:
what's this TODO?

Reply:
That's preexisting -- I assume @seesemichaelj put that there.  I assume it's about the fact that `key` is of type `any`, which should be narrowed down to something more specific.  You're seeing it anew due to it having moved to a different file.

Reply:
Or maybe I'll wait until the new decoder output format, actually.  In any case the point is I'll handle it eventually.